framework/scripts/wazuh_logtest.py

```diff
@@ -156,7 +156,7 @@ def main():
 
 
 class WazuhDeamonProtocol:
-    def __init__(self, version="1", origin_module="wazuh-logtest", module_name="wazuh-logtest"):
+    def __init__(self, version=1, origin_module="wazuh-logtest", module_name="wazuh-logtest"):
```

Comment:
We should change the docstring: `version (str, optional): protocol version. Defaults to "1".` -> `version (int, optional): protocol version. Defaults to 1.`